features/bevragen/autorisatie-gba.feature

```diff
@@ -0,0 +1,590 @@
+
+    @fout-case
+    Scenario: Zoeken met een optionele parameter waarvoor de afnemer niet geautoriseerd is
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)     | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 10210 10220 10230 10240 10310 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode |        |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+      | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | 19830526              | Maassen               | Pieter            | M                           |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam          | waarde                              |
+      | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+      | geslachtsnaam | Maassen                             |
+      | geboortedatum | 1983-05-26                          |
+      | geslacht      | M                                   |
+      | fields        | burgerservicenummer                 |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                 |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3            |
+      | title    | U bent niet geautoriseerd voor de gebruikte parameter(s).              |
+      | status   | 403                                                                    |
+      | detail   | U bent niet geautoriseerd voor het gebruik van parameter(s): geslacht. |
+      | code     | unauthorizedParameter                                                  |
+      | instance | /haalcentraal/api/brp/personen                                         |
+
+
+  Rule: Vragen met fields gevraagd om een veld waarvoor de gebruiker niet geautoriseerd is geeft een foutmelding
+    Om een veld te mogen vragen moet de afnemer geautoriseerd zijn voor alle LO BRP rubrieken waar het veld mee gevuld of van afgeleid wordt
+
+    @fout-case
+    Scenario: Afnemer vraagt om een veld (geboorte.plaats) waarvoor deze niet geautoriseerd is
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)     | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 10210 10220 10230 10240 10310 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode |        |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
```

Comment:
Moet je de geboorteplaats niet ook verstrekken in deze stap? Voor het resultaat maakt het natuurlijk niets uit maar wel voor het uitgangspunt.